src/use-swr.ts

```diff
@@ -268,7 +268,10 @@ function useSWR<Data = any, Error = any>(
   }
 
   const resolveIsValidating = () => {
-    return !!cache.get(keyValidating) || (key && willRevalidateOnMount())
+    return (
+      !!cache.get(keyValidating) ||
+      (key && willRevalidateOnMount() ? true : false)
```

Comment:
```suggestion
      (key && willRevalidateOnMount())
```

Reply:
```js
// In this way, the type of resolveIsValidating() would be boolean | ''
const resolveIsValidating = () => {
    return !!cache.get(keyValidating) || (key && willRevalidateOnMount())
}

// result = ''
const result = (() => {
    return !!false || ('' && false)
})()
```

Reply:
how about `!!(key && willRevalidateOnMount())` and can add a return type to resolveIsValidating?